website/blog/2023-04-14-release-0.14.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Release Notes - Podman Desktop 0.14
+description: Podman Desktop 0.14 has been released! 
+slug: podman-desktop-release-0.14
+authors: [deboer]
+tags: [podman-desktop, release, kubernetes, kind]
+hide_table_of_contents: false
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 0.14 - Our Kind-est release yet!
+
+We have been working on a Kind extension for a while now, and decided it is time to promote it
+into the release just in time for KubeCon and CloudNativeCon Europe!
+
+Some of these features were developed in the Kind extension over the last few releases,
+but since they are new to you, and they were not in previous release notes, we will do a
```

Comment:
```suggestion
but since they are now ready for production environments, and were not in previous release notes, we will do a
```

Reply:
Let's not call it "production environments". We also mentioned we wanted to put an "experimental" flag around the capabilities